sdk/core/Azure.Core/src/Shared/Utf8JsonDelayedRequestContent.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.IO;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Serialization;
+
+namespace Azure.Core
+{
+    internal class Utf8JsonDelayedRequestContent : RequestContent
```

Comment:
Is there a need to put this in shared source?  We're trying to reduce shared source, so if it's something needed by libraries, it might be best to make it public.

Reply:
We hadn't decided if we wanted this to be public yet, but it will be needed by all libraries